deps/wazuh_testing/wazuh_testing/db_interface/cve_db.py

```diff
@@ -128,3 +176,19 @@ def modify_nvd_metadata_vuldet(timestamp):
             sleep(1)
     else:
         raise OperationalError
+
+
+def check_inserted_value_exists(table, column, value):
+    """Function to check if a value exists in a specific column of a specific table.
```

Comment:
```suggestion
    """Check if a value exists in a specific table column.
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/2538/commits/762b15217466b07a5e4bfa23bcf2c647e1848760